src/main/java/com/forge/revature/controllers/PortfolioController.java

```diff
@@ -101,11 +106,162 @@ public Portfolio getByID(@PathVariable(name = "id") int id){
+		{
+			modifiedSubmissionTime = modifiedSubmissionTime.truncatedTo(ChronoUnit.DAYS);
+			modifiedSubmissionTime = modifiedSubmissionTime.plusDays(1);	
+		}    		
+		if(modifiedSubmissionTime.getDayOfWeek() == DayOfWeek.SUNDAY)
+		{
+			modifiedSubmissionTime = modifiedSubmissionTime.truncatedTo(ChronoUnit.DAYS);
+			modifiedSubmissionTime = modifiedSubmissionTime.plusDays(1);	
+		}
+		if(modifiedSubmissionTime.getHour() < 10)
+		{
+			modifiedSubmissionTime = modifiedSubmissionTime.truncatedTo(ChronoUnit.DAYS);
+			modifiedSubmissionTime = modifiedSubmissionTime.plusHours(10);
+		}else{
+	    	ZonedDateTime roundedSubmissionTime = modifiedSubmissionTime.truncatedTo(ChronoUnit.HOURS);
+	    	roundedSubmissionTime = roundedSubmissionTime.plusHours(1);
+			timeCounter = timeCounter.plusSeconds(roundedSubmissionTime.toEpochSecond() - modifiedSubmissionTime.toEpochSecond());
+			modifiedSubmissionTime = roundedSubmissionTime;
+		}
+	
+	
+		//while modifiedSubmissionTime time is less than modifiedReviewTime increment modifiedSubmissionTime time by one hour
+		//adding to the count if the time added is within work hours until it equals the modifiedReviewTime 
+		while(modifiedSubmissionTime.toEpochSecond() < modifiedReviewTime.toEpochSecond())
+		{
+			//skipping time outside of work hours which is okay because we made sure both are in work hours
+	    	if(modifiedSubmissionTime.getHour() > 17)
+	    	{
+	    		modifiedSubmissionTime = modifiedSubmissionTime.truncatedTo(ChronoUnit.DAYS);
+	    		modifiedSubmissionTime = modifiedSubmissionTime.plusDays(1);
+	    	}
+	    	if(modifiedSubmissionTime.getDayOfWeek() == DayOfWeek.SATURDAY)
+	    	{
+	    		modifiedSubmissionTime = modifiedSubmissionTime.truncatedTo(ChronoUnit.DAYS);
+	    		modifiedSubmissionTime = modifiedSubmissionTime.plusDays(1);	
+	    	}    		
+	    	if(modifiedSubmissionTime.getDayOfWeek() == DayOfWeek.SUNDAY)
+	    	{
+	    		modifiedSubmissionTime = modifiedSubmissionTime.truncatedTo(ChronoUnit.DAYS);
+	    		modifiedSubmissionTime = modifiedSubmissionTime.plusDays(1);	
+	    	}
+	    	if(modifiedSubmissionTime.getHour() < 10)
+	    	{
+	    		modifiedSubmissionTime = modifiedSubmissionTime.truncatedTo(ChronoUnit.DAYS);
+	    		modifiedSubmissionTime = modifiedSubmissionTime.plusHours(10);
+	    	}
+			//incrementing
+	    	while((modifiedSubmissionTime.toEpochSecond() < modifiedReviewTime.toEpochSecond())&&
+	    	(modifiedSubmissionTime.getHour() < 18))
+	    	{
+	    		timeCounter = timeCounter.plusSeconds(3600);
+	    		modifiedSubmissionTime = modifiedSubmissionTime.plusHours(1);
+	    	}
+		}
+		return timeCounter.getEpochSecond();
+    }
+
+	
+    
     @PostMapping("/{id}")
     public void updateUser(@PathVariable int id , @RequestBody Portfolio updated){
-        Optional<Portfolio> old = portRepo.findById(id);
-
+    	Optional<Portfolio> old = portRepo.findById(id);
+        
         if(old.isPresent()){
+            if(updated.isSubmissionTrigger() == true)
```

Comment:
This also seems like business logic that should be handled in the service layer.